account_move_tier_validation_approver/models/res_config_settings.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 ForgeFlow, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    require_approver_in_vendor_bills = fields.Boolean(
+        string="Require Approver In Vendor Bills",
+        help="Requires adding an approver before a vendor bill can be posted.",
+        related="company_id.require_approver_in_vendor_bills",
+        readonly=False,
+    )
+
+    def set_values(self):
+        tier_definition = self.company_id.validation_approver_tier_definition_id
+        if self.require_approver_in_vendor_bills:
+            if not tier_definition and not tier_definition.active:
```

Comment:
Non-blocking,
Will it better to use xml data to load the required tier.definition? to lesser code and as a noupdate, it can be more configurable.

Reply:
If you create it by xml, you are a bit constrained, aren't you? either you create one per company or a global one in the xml. After the latest changes I think the code is quite simple, we are always creating a tier def per company and just archiving or unarchiving it using the setting.

Could you check if it looks good to you now?

Thanks for reviewing as always :)